### PR TITLE
Add Task2 (Calculator 1.0)

### PR DESCRIPTION
**Pull request by ViktorDeinyi**
Support input double values. Available operators (+,  -,  /,  *,  %). Support re-entry of the operator in case of incorrect input.

### CODE REVIEW

File: Tasks/src/Calculator.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Calculator {
+
+        public static void main (String[] args) {
+            double sum, num1, num2;
+            Scanner scd = new Scanner(System.in);
```
Comment (Hmeliumm) [inline Tasks/src/Calculator.java]: You don't need 2 variables for Scanner (scd, scs).

File: Tasks/src/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            Scanner scs = new Scanner(System.in);
+            System.out.println("Давай попробуем что-то посчитать");
+            System.out.println("Введи первую цифру...");
+            num1 = scd.nextDouble();
+            System.out.print(num1);
+            System.out.println(" |Теперь оператор...");
+            String arg = scs.nextLine();
+            while (!arg.equals("+")&!arg.equals("-")&!arg.equals("*")&!arg.equals("/")&!arg.equals("%")) {
```
Comment (Hmeliumm) [inline Tasks/src/Calculator.java]: Please make it handle all String or remove

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Tasks/src/Calculator.java (hunk 1 above)
Comment (ViktorDeinyi) [reply Tasks/src/Calculator.java]: Ok, will be fixed in next PR.

File: Tasks/src/Calculator.java (hunk 2 above)
Comment (ViktorDeinyi) [reply Tasks/src/Calculator.java]: Did not understand the comment. Can you describe the problem in more detail?

**Comment by ViktorDeinyi** [comment]
New changes: "switch" instead of "if-else"; checking the correctness of entering numbers.

**Comment by ViktorDeinyi** [comment]
New changes: "switch" instead of "if-else"; checking the correctness of entering numbers.

**Review by Hmeliumm — CHANGES_REQUESTED** [review]
Switch-case calculator should be in separate PR

**Pull request merged**